templates/common/infra/bicep/core/host/container-apps-auth.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Container Apps Auth Config using Microsoft Entra as Identity Provider.'
+
+@description('The name of the container apps resource within the current resource group scope')
+param name string
+
+@description('The client ID of the Microsoft Entra application.')
+param clientId string
+
+@description('The name of the Container Apps secret that contains the client secret of the Microsoft Entra application.')
+param clientSecretName string
+
+@description('The OpenID issuer of the Microsoft Entra application.')
+param openIdIssuer string
+
+
+resource app 'Microsoft.App/containerApps@2023-05-01' existing = {
+  name: name
+}
+
+resource auth 'Microsoft.App/containerApps/authConfigs@2023-05-01' = {
+  parent: app
+  name: 'current'
+  properties: {
+    platform: {
+      enabled: true
+    }
+    globalValidation: {
+      redirectToProvider: 'azureactivedirectory'
+      unauthenticatedClientAction: 'RedirectToLoginPage'
+    }
+    identityProviders: {
+      azureActiveDirectory: {
+        registration: {
+          clientId: clientId
+          clientSecretSettingName: clientSecretName
+          openIdIssuer: openIdIssuer
+        }
+      }
+    }
+  }
+}
+
```

Comment:
This really just exposes a very small slice of the power of these auth configs.  I'd consider renaming this module to something more specific to entra/azuread since this is the only supported configuration here.